LICENSE-3RD-PARTY

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------------------------------------------
```

Comment:
[linguist-vendored](https://github.com/github/linguist#vendored-code) as appropriate

Reply:
Wasn't it @pentaphobe who introduced us to linguist-vendored in the first place??

Reply:
:giggle:

so I've really no excuse for that dangling CSS file...

The third party license file is at least partially manual